src/com/xilinx/rapidwright/placer/dreamplacefpga/DREAMPlaceFPGA.java

```diff
@@ -186,23 +186,42 @@ public static Design placeDesign(Design design, Path workDir) {
 
         // Load placed result
         Design placedDesign = null;
+        String interchangeResultFile = workDir.toString() + File.separator + "results/design/design";
         try {
-            placedDesign = PhysNetlistReader.readPhysNetlist(interchangeRootFile + Interchange.PHYS_NETLIST_EXT,
+            placedDesign = PhysNetlistReader.readPhysNetlist(interchangeResultFile + Interchange.PHYS_NETLIST_EXT,
                     design.getNetlist());
+            if (makeOutOfContext) {
+                placedDesign.setAutoIOBuffers(false);
+                placedDesign.setDesignOutOfContext(true);
+            }
         } catch (IOException e) {
             throw new UncheckedIOException(e);
         }
         return placedDesign;
     }
 
     public static void main(String[] args) {
-        if (args.length != 2 && args.length != 3) {
-            System.out.println("USAGE: <input DCP> <output DCP> [work_directory]");
+        // Usage: <input DCP> <output DCP> [--out_of_context] [work directory]
+        String MAKE_DCP_OUT_OF_CONTEXT = "--out_of_context";
+        if (args.length != 2 && args.length != 3 && args.length != 4) {
+            System.out.println("USAGE: <input DCP> <output DCP> [" 
+                            + MAKE_DCP_OUT_OF_CONTEXT + "] [work directory]");
             return;
         }
         Design input = Design.readCheckpoint(args[0]);
-        Path workDir = args.length == 3 ? Paths.get(args[2]) : Paths.get(System.getProperty("user.dir"));
-        Design placed = placeDesign(input, workDir);
+
+        boolean makeOutOfContext = false;
+        if (args.length >= 3) {
```

Comment:
What do I do to run the example if I don't want to use the `--out_of_context` option, but I do want to supply a work directory?